sandbox-backend/src/main/java/com/exadel/sandbox/team5/entity/Tag.java

```diff
@@ -0,0 +1,21 @@
+package com.exadel.sandbox.team5.entity;
+
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Setter
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+
+@Entity
+@Table(name = "tag")
+public class Tag extends BaseEntity {
+
+    @Column(name = "tagName", nullable = false, length = 50)
+    private String tagName;
```

Comment:
`tag` is redundant word because of class already has this information. Let's use `name`